### ISSUE TITLE
I'd love to help with automating the upload process

### ISSUE THREAD

**Issue by amalic** [CSSEGISandData/COVID-19#2161]
Today the daily update file had a different format. It seems very much to be a manual process.

I'd like to help to automate or semi-automate this process. Please feel free to send me a PM with contact details on my LinkedIn account which you can find at the bottom of my [Google Sheet](https://docs.google.com/spreadsheets/d/15OKNhZosR9xb0gShWtKw0P2toEN5AabZfp6kfnsVEKs)

**Comment by davidgs** [comment]
I have written a GO application that attempts to format the data into INfluxData Line-Protocol for ingestion into a Time Series Database, and I've also modified it to output to a standardized .csv file that includes lat,lng pairs -- reverse geocoded if necessary, etc. It even back-ports the csv format so that all files have the SAME csv format. Im desperately trying to keep it updated to the latest format changes.

**Comment by MelbourneDeveloper** [comment]
@amalic , @davidgs I agree with both of you. It's not really possible to get useful data from the CSV files by themselves. The formats have changed over time and some things don't reconcile.

Here is an SQLite database that stitches the data together as best as possible. Where the figures don't add up, I am reporting issues on this repo.

https://github.com/MelbourneDeveloper/COVID-19-DB
